maint_tools/convert_templates.py

```diff
@@ -1,18 +1,25 @@
+# /// script
+# requires-python = ">=3.10"
+# dependencies = [
+#   "nilearn[plotting]>=0.12",
+#    "PyQt6"
+# ]
+# ///
```

Comment:
FYI

this is a way to define the requirements for a script: https://peps.python.org/pep-0723/